packages/hoppscotch-app/newstore/localpersistence.ts

```diff
@@ -209,6 +213,54 @@ function setupSelectedEnvPersistence() {
   })
 }
 
+function setupWebsocketPersistence() {
+  const request = JSON.parse(
+    window.localStorage.getItem("WebsocketRequest") || "null"
+  )
+
+  setWSRequest(request)
+
+  WSRequest$.subscribe((req) => {
+    window.localStorage.setItem("WebsocketRequest", JSON.stringify(req))
+  })
```

Comment:
only the request is saved. same for the others.

Reply:
aaaah. missed that. sorry 😅